aiven/client/cli.py

```diff
@@ -560,13 +622,18 @@ def service__types(self):
                         default_desc = "(default={!r})".format(default) if default is not None else ""
                         description = ": {}".format(spec["description"]) if "description" in spec else ""
                         types = spec["type"]
-                        if not isinstance(types, list):
-                            types = [types]
-                        type_str = " or ".join(t for t in types if t != "null")
-                        print("  {title}{description}\n"
-                              "     => -c {name}=<{type}>  {default}"
-                              .format(name=name, type=type_str,
-                                      default=default_desc, title=spec["title"], description=description))
+                        if isinstance(types, str) and types == "null":
```

Comment:
Should first convert to list and then print the `--remove-option` option in case any of the list values is `"null"` so that this works also for data types other than object.

Reply:
Then there's problem that all options accepting null as value will be reported for both add/remove.

Reply:
Shouldn't they be? Maybe the output format could be made more compact if it gets too cluttered. Like `(Use -d {name} to remove)` after the `-c {name}=<{type}>  {default}` part.